### DISCUSSION TITLE
Radeon HD 6870 MacOs10.5.7 Catalina?

### DISCUSSION THREAD

**Discussion by Bogdis** [chris1111/Legacy-Video-patch#7] [Q&A]
Hello and thanks for this great package.
I´ve got my hd 6870 running in Catalina now but it only gets recognised as hd 6xxx 1gb and I get weird glitches and a bit sluggish.
do you have any idea if there is something I can do?
I've tried to get the intel hd 3000 working but can't get it to work get sip panic after install of the patch and after a ton of research I managed to boot in to Catalina but not with full res.
Thanks

**Comment by chris1111** [comment]
Radeon HD is not full support but HD 3000 Yes
If you have try another things before using legacy video patch this is not work What is the Bootloader ? Put the EFI Folder on ZIP here

**Comment by Bogdis** [comment]
Hi, I understand that 6870 is not full supported but in your video the radeon card shows upp as HD6670 but mine only show 6xxx and sluggish effects. I don't need full support only using it for surfing and Plex server only light stuff. 
I´ve installed it several times with nothing else installed and every time I'm getting sip panic. 
Then I saw that it works for radeon cards to so I tried with that one and it worked but the hd 3000 won't work at all.
I use UniBeast with clover.
Did a clean install a couple day ago with the hd6870 and didn't even tried the hd3000 to se if it works but I did tried it now and I can boot in to it but only with the resolution 1280x720 and in about this Mac it shows up as graphics display 3mb or something like that forgot to take a screenshot.

**Comment by chris1111** [comment]
What is the hardware you have exactly ?
for the ATI card you need inject ATi

**Reply by Bogdis** [reply]
I have an i5 2500k with a Ga-z68x-ud3h-b3
tried inject ati to but didn't work do I need something else?

**Comment by chris1111** [comment]
Wait Radeon 6870 is not support in macOS

**Reply by Bogdis** [reply]
Yes i know that its not supported but with your patch it should work right? Your 6670 card worked why is mine not working properly?

**Comment by chris1111** [comment]
for making HD 3000 working you need to remove the Radeon card

**Reply by Bogdis** [reply]
I will try removing the card and see if that helps, but like i said i tested before and but didint get the right resolution and the graphics is not recognized properly

**Comment by chris1111** [comment]
What is the specification of the Hackintosh
post the system Repport About this Mac here on zip

**Reply by Bogdis** [reply]
Hi, I´ve just removed the Radeon card but still the same resolution at 1024x768 and the about this Mac shows Display 3mb.
Do I need to install whatevergreen to?

**Reply by chris1111** [reply]
> Hi, I´ve just removed the Radeon card but still the same resolution at 1024x768 and the about this Mac shows Display 3mb.
> Do I need to install whatevergreen to?
> 
> [Archive.zip](https://github.com/chris1111/Legacy-Video-patch/files/5720114/Archive.zip)

What is the specification of the Hackintosh

This is AMD card not Intel
![Screenshot 2020-12-19 at 21 37 36](https://user-images.githubusercontent.com/6248794/102700384-e1cfb180-421a-11eb-83cc-f602259c5e9b.png)

**Reply by Bogdis** [reply]
Yes that's right I've attach both one with hd6870 and one with hd3000 (look at the other screenshot for intel hd 3000)
My Hackintosh in i5 2500k with a Ga-z68x-ud3h-b3

**Comment by chris1111** [comment]
Ga-z68x-ud3h-b3 
You need to Set Up the **IGPU** in bios and the Vram at 512 MB

**Reply by Bogdis** [reply]
Noppe does not work 🙁
why does it say that no kext is loaded under system rapport and graphics?

**Reply by chris1111** [reply]
> Noppe does not work 🙁
> why does it say that no kext is loaded under system rapport and graphics?


Kext are not loading

Use this config.plist for Intel HD 3000

[config.plist.zip](https://github.com/chris1111/Legacy-Video-patch/files/5720234/config.plist.zip)
Reboot

**Reply by Bogdis** [reply]
Thank you 🙏 it worked 😁😁😁 
One more question can i just move my smbios info to the new config file?

**Reply by chris1111** [reply]
> Thank you 🙏 it worked 😁😁😁
> One more question can i just move my smbios info to the new config file?

😀 Its your config.plist , SMBIOS is the same Its mod by me I ad this on your config
![Graphics](https://user-images.githubusercontent.com/6248794/102701309-cd43e700-4223-11eb-9431-07decfb22e02.png)

![FakeID](https://user-images.githubusercontent.com/6248794/102701310-d1700480-4223-11eb-99fe-2825043718be.png)

I will Update the Page for Hackintosh User PC 🤙 
Thanks for your patience

**Reply by Bogdis** [reply]
Aha ok great thank you very much 😁

**Comment by chris1111** [comment]
Its your config.plist I not touch to SMBIOS section I add this 
![Graphics](https://user-images.githubusercontent.com/6248794/102701440-e0a38200-4224-11eb-9eda-6c4022780b61.png)

![FakeID](https://user-images.githubusercontent.com/6248794/102701445-e7ca9000-4224-11eb-8e71-4ea5e8ca7c68.png)

Thanks for you patience  🤙 😀 


Post a picture about this Mac  

and Please post result here
https://www.tonymacx86.com/threads/full-acceleration-for-intel-hd-3000-and-older-nvidia-graphics-in-macos-catalina.290488/

**Reply by Bogdis** [reply]
![Screenshot 2020-12-20 at 00 12 27](https://user-images.githubusercontent.com/48528615/102701594-d69b7680-4258-11eb-8d5b-6914ecd798a8.png)

**Comment by chris1111** [comment]
Thank you 😎

I add this to the first page
https://github.com/chris1111/Legacy-Video-patch/discussions/9

**Comment by chris1111** [comment]
Hi Bogdis do you have glitchy graphics some times with your Intel HD 3000? 
EDIT ***
Do you find that your macOS Catalina works well with the Patch?

**Reply by Bogdis** [reply]
Hi, I'm using it over Remote Desktop and yes is glitchy and choppy and slow sometimes.
But I need to test with a display first to se if its the same or not, but I was using Sierra before this over Remote Desktop with radeon hd 6870 and it was much better.

**Reply by chris1111** [reply]
> Hi, I'm using it over Remote Desktop and yes is glitchy and choppy and slow sometimes.
> But I need to test with a display first to se if its the same or not, but I was using Sierra before this over Remote Desktop with radeon hd 6870 and it was much better.

OK But now it would not be better with the Radeon HD it would be much worse with lags without really having the ful QE/CI
The graphics glitch with HD 3000 its a known issue even with a real Mac  you can try add the boot args **dart=0**

**Reply by Bogdis** [reply]
Yes it was much worse with the radeon card. I´m already using the boot flag. This is not my main system (just bought my first Mac mini the new M1) so it doesn't really matter, but I´m happy never the less with this to:). I´m going to move over to a nas system soon. Like I said before its mainly for Plex server.

**Comment by chris1111** [comment]
Ha ok anyway the old Z68 allowed you to install macOC for a long time already and continues to provide good service 😁

**Reply by Bogdis** [reply]
Yes its been quite some time, I´ve been using this system for like 9 years now and its still going :)

**Comment by chris1111** [comment]
With a cheap Geforce 710 the Z68 still work great in macOS Big Sur

**Reply by Bogdis** [reply]
ok, if I buy that card do I need to do a new install on Catalina or just plug it in and that's it?

**Comment by chris1111** [comment]
You need to use the new Clover OC Integration  for Big Sur  or using Opencore 
anyway this is not the place to talk about that you can find tutorial on Tony M  for Big Sur

**Reply by Bogdis** [reply]
Thank you for all the help and tips. Merry Christmas 🎄 to you

**Reply by chris1111** [reply]
> Thank you for all the help and tips. Merry Christmas 🎄 to you

Merry Christmas  to you and all your family 😀
![33e0b76968539b056574a91ba8beacda](https://user-images.githubusercontent.com/6248794/102907167-a401a180-4443-11eb-9a0a-67db769b4b41.gif)
